src/pytest_plugins/execute/eth_config/eth_config.py

```diff
@@ -0,0 +1,48 @@
+"""Pytest plugin to test the `eth_config` RPC endpoint in a node."""
+
+import pytest
+from pydantic import TypeAdapter
+
+from ethereum_test_forks import Fork
+from ethereum_test_rpc import EthRPC
+
+COMMAND_EXAMPLE = """
+uv run execute eth-config --chain-id 0x88bb0
+ --current Prague --current-activation-time 0 --next Osaka --next-activation-time 1742999832
+ --next-blob-schedule '{"baseFeeUpdateFraction": 5007716, "max": 9, "target": 6}'
+"""
+
+ForkAdapter = TypeAdapter(Fork)  # type: ignore
+
+
+def pytest_addoption(parser):
+    """Add command-line options to pytest."""
+    eth_config_group = parser.getgroup("execute", "Arguments defining eth_config test behavior.")
+    eth_config_group.addoption(
+        "--network",
+        action="store",
+        dest="network",
```

Comment:
Can we fail early if this value is invalid by checking it's availability in the available network configs in `pytest_configure()`?